modules/siddhi-query-api/src/main/java/org/wso2/siddhi/query/api/definition/TableDefinition.java

```diff
@@ -44,6 +44,17 @@ public TableDefinition annotation(Annotation annotation) {
         return this;
     }
 
+    public void replaceAnnotation(Annotation annotation) {
```

Comment:
Fixed in 2333816.

Changed the functionality to removeAnnotation, this is needed to remove the old annotation(store) in the definition